### ISSUE TITLE
Inject global-attributes into antora.yml File?

### ISSUE THREAD

**Issue by bai-yi-bai** [redpanda-data/docs-extensions-and-macros#63]
Hi, 

Is it possible to pass the global-attributes to the antora.yml? I would like to use them with Antora Assembler and use them on our custom title page generator, and for the running-content. Currently, they come through as [object Object].

Copied from:
https://antora.zulipchat.com/#narrow/stream/282400-users/topic/Get.20asciidoc.20attributes.20values.20from.20external.20sources/near/451426074

**Comment by JakeSCahill** [comment]
Hey @bai-yi-bai did you manage to resolve this?

**Comment by bai-yi-bai** [comment]
Hi, I followed up a few days later on the Antora Zulipchat in another thread.

https://antora.zulipchat.com/#narrow/channel/282400-users/topic/.E2.9C.94.20Feedback.20and.20Advice.20on.20my.20add-document-attributes.20Exten.2E.2E.2E

 Although I did mark the topic as complete, I didn't implement this approach in our workflow and I think I have some subsequent code updates. It's been a few months and other more pressing matters came up, but I think I was able to implement the reverse-lookup. I do plan to share the code, but I need to carve out the time.
 
 Again, I want to say thanks to you and redpanda for creating these extensions. They serve as an invaluable tool for learning how to write Antora extensions. Thanks to them, I feel I have a much better understanding of how Antora works.
